streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/pages/setting/AlarmPage.java

```diff
@@ -0,0 +1,336 @@
+            AlarmPageDingTalkSetting alarmPageDingTalkSetting = (AlarmPageDingTalkSetting) faultAlertSetting;
+            inputDingTalkURL.sendKeys(alarmPageDingTalkSetting.url());
+            inputDingTalkToken.sendKeys(alarmPageDingTalkSetting.token());
+            inputDingTalkReceiveUser.sendKeys(alarmPageDingTalkSetting.receiveUser());
+            // if open the secret button, fill token value.
+            if (alarmPageDingTalkSetting.isSecretEnable()) {
+                btnDingTalkSecret.click();
+                inputDingTalkSecretToken.sendKeys(alarmPageDingTalkSetting.secretToken());
+            }
+            if (alarmPageDingTalkSetting.isEffectToAllUsers()) {
+                btnDingTalkEffectToAllUsers.click();
+            }
+        }
+    }
+
+    private class CreateWeChatAlertForm implements AlertService {
+
+        CreateWeChatAlertForm() {
+            PageFactory.initElements(driver, this);
+        }
+
+        @FindBy(id = "form_item_weToken")
+        private WebElement inputWeChatToken;
+
+        @Override
+        public void createAlert(FaultAlertSetting faultAlertSetting) {
+            AlarmPageWeChatSetting alarmPageWeChatSetting = (AlarmPageWeChatSetting) faultAlertSetting;
+            inputWeChatToken.sendKeys(alarmPageWeChatSetting.token());
+        }
+    }
+
+    private class CreateLarkAlertForm implements AlertService {
+
+        CreateLarkAlertForm() {
+            PageFactory.initElements(driver, this);
+        }
+
+        @FindBy(id = "form_item_larkToken")
+        private WebElement inputLarkToken;
+
+        @FindBy(id = "form_item_larkIsAtAll")
+        private WebElement btnLarkEffectToAllUsers;
+
+        @FindBy(id = "form_item_larkSecretEnable")
+        private WebElement btnLarkSecret;
+
+        @FindBy(id = "form_item_larkSecretToken")
+        private WebElement inputLarkSecretToken;
+
+        @Override
+        public void createAlert(FaultAlertSetting faultAlertSetting) {
+            AlarmPageLarkSetting alarmPageLarkSetting = (AlarmPageLarkSetting) faultAlertSetting;
+            inputLarkToken.sendKeys(alarmPageLarkSetting.token());
+            // if open the secret button, fill token value.
+            if (alarmPageLarkSetting.isSecretEnable()) {
+                btnLarkSecret.click();
+                inputLarkSecretToken.sendKeys(alarmPageLarkSetting.SecretToken());
+            }
+            if (alarmPageLarkSetting.isEffectToALlUsers()) {
+                btnLarkEffectToAllUsers.click();
+            }
+        }
+    }
+
+    // not visible here.
+    private class CreateSMSAlertForm implements AlertService {
+
+        CreateSMSAlertForm() {
+            PageFactory.initElements(driver, this);
+        }
+
+        @Override
+        public void createAlert(FaultAlertSetting faultAlertSetting) {
+        }
+    }
+
+    private interface AlertService {
+
+        void createAlert(FaultAlertSetting faultAlertSetting);
+    }
```

Comment:
We don't need this since this is a test. Not a service. For example of dynamic form parameters, you can refer to `FlinkSQL116OnYarnTest` or `Flink116OnYarnClusterDeployTest`.